.kokoro/build-with-run.sh

```diff
@@ -44,7 +44,8 @@ gcloud config set project $GOOGLE_CLOUD_PROJECT
 # Version is in the format <PR#>-<GIT COMMIT SHA>.
 # Ensures PR-based triggers of the same branch don't collide if Kokoro attempts
 # to run them concurrently. Defaults to 'latest'.
-export SAMPLE_VERSION="${KOKORO_GIT_COMMIT:-latest}"
+RAW_SAMPLE_VERSION="${KOKORO_GIT_COMMIT:-latest}"
+export SAMPLE_VERSION="${RAW_SAMPLE_VERSION:0:15}"
```

Comment:
nit: add comment about shortening the sample version to fit cloud run limitations

Reply:
This change is not related to Cloud Run, but was a blind experiment to see if GCR had a limitation. It seemed to help, 🤷